applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/eigenfrequency_response_function_utility.h

```diff
@@ -314,33 +315,42 @@ class EigenfrequencyResponseFunctionUtility
                         gradient_contribution[coord_dir_i] += gradient_prefactors[i] * inner_prod(eigenvectors_of_element[i] , aux_vector) * mWeightingFactors[i];
                     }
 
-                   node_i.GetInitialPosition()[coord_dir_i] -= mDelta;
+                    node_i.GetInitialPosition()[coord_dir_i] -= mDelta;
```

Comment:
noalias

Reply:
Why do i need the noalias if i assign a value inside the vector?
I thought its only necessary for matrix/vector operations..

Reply:
Ok found it on google: Apparently passing by reference does not give a benefit for types like int (maybe even produces overhead).

Reply:
the InitialCoordinates are an array3, there you could/should also use it